### DISCUSSION TITLE
Is it possible to launch multi-node runs without pdsh?

### DISCUSSION THREAD

**Discussion by kyleliang919** [deepspeedai/DeepSpeed#2819] [Q&A] [answered]
I am on a slurm system and I don not have sudo access to install pdsh? Is there a way to use deepspeed to run multiple node without the launcher?

**Comment by wzj423** [comment, marked as answer]
Yes. Simply use `torchrun` instead of `deepspeed`, see this example here. https://github.com/microsoft/DeepSpeed/issues/1603#issuecomment-984308513
